cost/azure/old_snapshots/azure_delete_old_snapshots.pt

```diff
@@ -0,0 +1,225 @@
+name "Azure Old Snapshots"
+rs_pt_ver 20180301
+type "policy"
+short_description "Checks for snapshots older than specified number of days and, optionally, deletes them. See the [README](https://github.com/flexera/policy_templates/tree/master/cost/azure/old_snapshots) and [docs.rightscale.com/policies](https://docs.rightscale.com/policies/) to learn more."
+severity "low"
+category "Cost"
+info(
+  version: "2.0",
+  provider: "Azure",
+  service: "Storage",
+  policy_set: "Old Snapshots"
+  )
+
+##################
+# User inputs    #
+##################
+
+parameter "param_email" do
+  type "list"
+  label "Email addresses of the recipients you wish to notify"
```

Comment:
missing description.  the label should be simple and short.

Reply:
Added description and changed label